src/Altinn.ApiClients.Maskinporten/Altinn.ApiClients.Maskinporten.csproj

```diff
@@ -2,9 +2,15 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <Version>0.9.62-alpha</Version>
+    <Version>1.0.0</Version>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PackageId>Altinn.ApiClients.Maskinporten</PackageId>
```

Comment:
Should we include <Company> tag and set it to Digitaliseringsdirektoratet (or Digdir) and perhaps a short description of what the package does using the <Description> tag

Reply:
Yes, this should probably be standardized across all packages. I'll try to come up with a suggestion.